samples/ImageList/tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "noImplicitAny": true,
     "skipLibCheck": true,
     "declaration": true,
-    "jsxFactory": "React",
+    "jsxFactory": "React.createElement",
```

Comment:
This change seems to have been necessary when I was using the "tsc" workaround @a-tarasyuk was complaining about. The TS compiler would generate:
    `return React(RootView, null)`
instead of
    `return React.createElement(RootView, null)`

Without the workaround it seems the babel/whatever pipeline doesn't take this jsxFactory property from tsconfig.json into account.
I could change back, but it seems the new version is more correct since it helps those using tsc. I admit I don't know why the setting is there in the first place.

I did test start:ios, start:web, and now start:windows, all with good results. I just didn't do Android cause I'm avoiding that one.